tfe/resource_tfe_team.go

```diff
@@ -24,26 +27,60 @@ func resourceTFETeam() *schema.Resource {
 				Required: true,
 				ForceNew: true,
 			},
-
 			"organization": {
 				Type:     schema.TypeString,
 				Required: true,
 				ForceNew: true,
 			},
+			"organization_access": {
+				Type:     schema.TypeList,
+				MaxItems: 1,
+				Optional: true,
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						"manage_policies": {
+							Type:     schema.TypeBool,
+							Optional: true,
+						},
+						"manage_workspaces": {
+							Type:     schema.TypeBool,
+							Optional: true,
+						},
+						"manage_vcs_settings": {
+							Type:     schema.TypeBool,
+							Optional: true,
+						},
+					},
+				},
+			},
+			"visibility": {
+				Type:     schema.TypeString,
+				Optional: true,
+				Computed: true,
+				ValidateFunc: validation.StringInSlice([]string{
+					"secret",
+					"organization",
+				}, false),
+			},
 		},
 	}
 }
 
 func resourceTFETeamCreate(d *schema.ResourceData, meta interface{}) error {
 	tfeClient := meta.(*tfe.Client)
 
-	// Get the name and organization.
+	// Get team attributes.
 	name := d.Get("name").(string)
 	organization := d.Get("organization").(string)
 
 	// Create a new options struct.
 	options := tfe.TeamCreateOptions{
-		Name: tfe.String(name),
+		Name:               tfe.String(name),
+		OrganizationAccess: getTeamOrganizationAccess(d),
```

Comment:
Given that this is also an optional block, I think this might be clearer to follow the same `if v, ok := d.GetOk("organization_access"); ok {` pattern; that's what I did in [my version of this](https://github.com/terraform-providers/terraform-provider-tfe/pull/52/files#diff-d0f8a2703c6d17783be2da858fb5047dR77-R86), too.